src/main/java/net/mcreator/ui/blockly/BlocklyJavascriptBridge.java

```diff
@@ -382,6 +359,37 @@ public void addExternalTrigger(ExternalTrigger external_trigger) {
 		return retval.toArray(new String[0]);
 	}
 
+	public static String[] getDataListFieldValues(Workspace workspace, String datalist, String typeFilter,

```

Comment:
Given this method is only used by tests, maybe it could be a private method in GTProcedureBlocks?